### PR TITLE
Add customer-facing docs

### PR DESCRIPTION
**Pull request by matthewarthur**
PR adds a new page to our customer docs explicitly outlining our shared responsibility model., 

### Motivation

Internal review identified the need for an SRM docs page.

### Tips for reviewer

Should only modify /docs by adding a new markdown page.

### CODE REVIEW

File: doc/user/content/customer-responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+----
+title: "Customer Responsibility Model"
```
Comment (morsapaes) [inline doc/user/content/customer-responsibilities.md:10]:
```suggestion
title: "Customer responsibility model"
```

**Review by morsapaes — APPROVED** [review]

File: doc/user/content/customer-responsibilities.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Summary
+
+The Materialize operational data warehouse is built with a shared responsibility model to ensure the highest levels of data integrity, availability, and resilience. This document outlines the specific responsibilities of customers to manage their data and connectivity effectively. Understanding and fulfilling these responsibilities is critical to leveraging the full potential of our service.
+
+## Details
+
+### Backup
+
+As part of our shared responsibility model, customers are tasked with maintaining backups of their original data. While our service provides robust data processing capabilities, ensuring that you have a comprehensive backup strategy is crucial. This includes:
```
Comment (benesch) [inline doc/user/content/customer-responsibilities.md:26]:
```suggestion
As part of our shared responsibility model, customers are tasked with maintaining backups of their original data. While the platform provides robust data processing capabilities, ensuring that you have a comprehensive backup strategy is crucial. This includes:
```

File: doc/user/content/customer-responsibilities.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- **Data Integrity**: Verifying the integrity of backups to ensure data is complete and accurate, enabling effective recovery if needed.
+- **Secure Storage**: Utilizing secure and reliable storage solutions to protect your backups from unauthorized access and potential data loss scenarios.
+
+Materialize maintains backups of core system state and ingested data to ensure timely recovery in the event of an outage. The data of record is expected to be maintained in upstream customer systems.
+
+### Recovery
+
+Responsibility for recovery from data connectivity issues rests with the customer. In the event of an outage or disruption, customers are expected to:
+
+- **Connection Recovery**: Re-establish connections to our SaaS service promptly to minimize downtime.
```
[the remark this answers is not in the document]
Comment (benesch) [reply doc/user/content/customer-responsibilities.md:38]: Using the format "**A bold heading.** An explanation in sentence case." matches the style we use elsewhere in the docs.

**Review by benesch — COMMENTED** [review]
Nice! Thanks for putting this together. A few inline comments.

**Comment by morsapaes** [comment]
Sorry, I was on mobile yesterday and didn't do a proper review. Applied Nikhil's suggestions and a few other minor edits. I noticed that we mention our Privacy Police throughout the website often without linking to it. @nisarhassan12

**Review by benesch — APPROVED** [review]
LGTM!

**Pull request merged**